src/Accordion/accordion.container.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import { Container } from 'unstated';
+
+export type Item = {
+    uuid: string | number,
+    expanded: boolean,
+    disabled: boolean,
+};
+export type StoreState = {
+    items: Array<Item>,
+    accordion: boolean,
+    onChange: Function,
+};
+
+class AccordionContainer extends Container<StoreState> {
+    state = {
+        items: [],
+        accordion: true,
+        onChange: () => {},
+    };
+
+    init(accordion: boolean, onChange: Function) {
```

Comment:
I was going to suggest refactoring this to:
```
    init({ state }: $SubType<StoreState>) {
    	this.setState(state);
    }
```
But then you've really only just created an alias for 'setState', so you might as well just remove this and use `setState` (or `setOnChange` and `setAccordion`) instead.